### PR TITLE
Add Kavenegar.com, Iranian API based sms service

### PR DESCRIPTION
**Pull request by vahit**
Hi, I've added support for kavenegar.com (Iranian API based sms service) because twillo and messagebird banned iranian users. I've copied all of code from messagebird.com plugin.

### CODE REVIEW

File: conf.d/health_alarm_notify.conf [hunk 1]
```diff
@@ -205,6 +206,28 @@ DEFAULT_RECIPIENT_MESSAGEBIRD=""
 
 
 #------------------------------------------------------------------------------
+# Kavenegar (Kavenegar.com) SMS options
+
+# multiple recipients can be given like this:
+#                  "09155555555 09177777777"
+
+# enable/disable sending messagebird SMS
```
Comment (ktsaou) [inline conf.d/health_alarm_notify.conf]: please replace `messagebird` with `kavenegar`.

File: plugins.d/alarm-notify.sh [hunk 2]
```diff
@@ -270,6 +271,12 @@ MESSAGEBIRD_NUMBER=
 DEFAULT_RECIPIENT_MESSAGEBIRD=
 declare -A role_recipients_messagebird=()
 
+# kavenegar configs
+KAVENEGAR_API_KEY=""
+KAVENEGAR_SENDER=""
+DEFAULT_RECIPIENT_KAVENEGAR=""
+declare -A role_recipients_kavenegar=""
```
Comment (ktsaou) [inline plugins.d/alarm-notify.sh]: Please set it like this:

```sh
declare -A role_recipients_kavenegar=()
```

**Comment by ktsaou** [comment]
Seems good. Thanks!

Please also provide a wiki page for it, like the following:

image

**Comment by vahit** [comment]
And all done. Wiki page [URL](https://github.com/firehol/netdata/wiki/kavenegar-notifications)

**Comment by ktsaou** [comment]
merged! thanks!

**Pull request merged**